extensions/CocoStudio/Reader/SceneReader.h

```diff
@@ -62,8 +62,11 @@ class CC_EX_DLL SceneReader
 {
 
 public:
-	static SceneReader* sharedSceneReader();
-	static void purge();
+	CC_DEPRECATED_ATTRIBUTE static SceneReader* sharedSceneReader();
+	static SceneReader* getInstance();
+	CC_DEPRECATED_ATTRIBUTE static void purge();
+	static void destroyInstance();
+	static void destroyCocosStudio();
```

Comment:
@xiaofeng11, Yes, I suggested cocostudio needs to provide an interface to destroy all resources studio used. Please look at my email, I said an interface like `destroyCocosStudio`, but it isn't that you have to use the name.

BTW, I didn't say to put this method in `SceneReader`, I think you guys design CocosStudio, you have the final decision to the function name and where the function placed.

It's better to add a new file to provide this function. For example,
Now we have CocoStudio.h, probably we need `CocoStudio.cpp`
And put that function to there.

@minggo what's your opinion?